ush/load_modules_run_task.sh

```diff
@@ -92,7 +92,7 @@ jjob_fp="$2"
 #
 machine=$(echo_lowercase $MACHINE)
 
-source "${SR_WX_APP_TOP_DIR}/etc/lmod-setup.sh"
+source "${SR_WX_APP_TOP_DIR}/etc/lmod-setup.sh" ${machine}
```

Comment:
Does it need to be this (quotation mark after ${machine}):

source "${SR_WX_APP_TOP_DIR}/etc/lmod-setup.sh ${machine} "

It may work both ways though

Reply:
It works both ways but having the machine variable outside of the quotes removes this warning `".../lmod-setup.sh gaea not found"` at the top of the log files.